zerver/views/user_settings.py

```diff
@@ -25,7 +25,6 @@
 from zerver.lib.request import JsonableError
 from zerver.models import UserProfile, Realm, name_changes_disabled
 
-@authenticated_json_post_view
```

Comment:
Probably not needed, see https://github.com/zulip/zulip/pull/2822#issuecomment-268554315